src/fu-util.c

```diff
@@ -2226,65 +2225,12 @@ fu_util_downgrade (FuUtilPrivate *priv, gchar **values, GError **error)
 static gboolean
 fu_util_hwids (FuUtilPrivate *priv, gchar **values, GError **error)
 {
-	g_autoptr(FuSmbios) smbios = fu_smbios_new ();
-	g_autoptr(FuHwids) hwids = fu_hwids_new ();
-	const gchar *hwid_keys[] = {
-		FU_HWIDS_KEY_BIOS_VENDOR,
-		FU_HWIDS_KEY_BIOS_VERSION,
-		FU_HWIDS_KEY_BIOS_MAJOR_RELEASE,
-		FU_HWIDS_KEY_BIOS_MINOR_RELEASE,
-		FU_HWIDS_KEY_MANUFACTURER,
-		FU_HWIDS_KEY_FAMILY,
-		FU_HWIDS_KEY_PRODUCT_NAME,
-		FU_HWIDS_KEY_PRODUCT_SKU,
-		FU_HWIDS_KEY_ENCLOSURE_KIND,
-		FU_HWIDS_KEY_BASEBOARD_MANUFACTURER,
-		FU_HWIDS_KEY_BASEBOARD_PRODUCT,
-		NULL };
-
-	/* read DMI data */
-	if (!fu_smbios_setup (smbios, error))
-		return FALSE;
-	if (!fu_hwids_setup (hwids, smbios, error))
-		return FALSE;
-
-	/* show debug output */
-	g_print ("Computer Information\n");
-	g_print ("--------------------\n");
-	for (guint i = 0; hwid_keys[i] != NULL; i++) {
-		const gchar *tmp = fu_hwids_get_value (hwids, hwid_keys[i]);
-		if (tmp == NULL)
-			continue;
-		g_print ("%s: %s\n", hwid_keys[i], tmp);
-	}
-
-	/* show GUIDs */
-	g_print ("\nHardware IDs\n");
-	g_print ("------------\n");
-	for (guint i = 0; i < 15; i++) {
-		const gchar *keys = NULL;
-		g_autofree gchar *guid = NULL;
-		g_autofree gchar *key = NULL;
-		g_autofree gchar *keys_str = NULL;
-		g_auto(GStrv) keysv = NULL;
-		g_autoptr(GError) error_local = NULL;
-
-		/* get the GUID */
-		key = g_strdup_printf ("HardwareID-%u", i);
-		keys = fu_hwids_get_replace_keys (hwids, key);
-		guid = fu_hwids_get_guid (hwids, key, &error_local);
-		if (guid == NULL) {
-			g_print ("%s\n", error_local->message);
-			continue;
-		}
-
-		/* show what makes up the GUID */
-		keysv = g_strsplit (keys, "&", -1);
-		keys_str = g_strjoinv (" + ", keysv);
-		g_print ("{%s}   <- %s\n", guid, keys_str);
-	}
-
-	return TRUE;
+	const gchar *fn = g_build_filename (LIBEXECDIR, "fwupd", "fwupdtool", NULL);
+	g_set_error (error,
+		     FWUPD_ERROR,
+		     FWUPD_ERROR_NOT_SUPPORTED,
+		     "This functionality has moved to %s", fn);
+	return FALSE;
```

Comment:
Is there a lot of documentation that says to use fwupdmgr for this? I know we moved several things to fwupdtool without messages like this at least

Reply:
Fixed.

Reply:
did you forget to push?

Reply:
Indeed, Friday! Pushed now.